crates/cargo-test-macro/src/lib.rs

```diff
@@ -255,7 +257,16 @@ fn has_rustup_stable() -> bool {
         // https://github.com/rust-lang/rustup/issues/3036 is resolved.
         return false;
     }
-    check_command("cargo", &["+stable", "--version"])
+    // Cargo mucks with PATH on Windows, adding sysroot host libdir, which is
+    // "bin", which circumvents the rustup wrapper. Use the path directly from
+    // CARGO_HOME.
```

Comment:
Would <https://github.com/rust-lang/cargo/pull/13468> fix this issue?

Reply:
Maybe! It will need to wait until that hits stable, though.

Reply:
Oh, yeah. We need it in stable :)